packages/style-engine/phpunit/class-wp-style-engine-test.php

```diff
@@ -133,22 +133,6 @@ public function data_get_styles_fixtures() {
 				),
 			),
 
-			'invalid_context'                              => array(
-				'block_styles'    => array(
-					'color'   => array(
-						'text' => 'var:preset|color|sugar',
-					),
-					'spacing' => array(
-						'padding' => '20000px',
-					),
-				),
-				'options'         => array(
-					'convert_vars_to_classnames' => true,
-					'context'                    => 'i-love-doughnuts',
-				),
-				'expected_output' => array(),
-			),
-
```

Comment:
Again, just confirming I'm following along: this test is no longer needed because we're no longer allow-listing the context? (We're effectively passing along the `context` as the store name, now, allowing any string-based value as the store name).

Reply:
Exactly! Thanks for clarifying.